.expeditor/scripts/shared.sh

```diff
@@ -40,148 +19,20 @@ curlbash_hab() {
     # are the same. let's just delete it
     rm -rf /hab/pkgs/core/hab/0.82.0
     curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash -s -- -t "$pkg_target"
+    case "${pkg_target}" in
+        x86_64-linux)
```

Comment:
These cases can be combined; it makes it clearer that they do the same thing
```suggestion
        x86_64-linux | x86_64-linux-kernel2)
```
or
```suggestion
        x86_64-linux*)
```